name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/StatementOfAssetsViewer.java

```diff
@@ -746,6 +747,27 @@ private void addDividendColumns(List<ReportingPeriod> options)
         column.setSorter(ColumnViewerSorter.create(new ElementComparator(labelProvider)));
         column.setVisible(false);
         support.addColumn(column);
+
+        // dividend yield on costs auf der hauptübersicht/vermögensaufstellung
+        column = new Column("yocdiv", Messages.ColumnDividendYieldOnCost, SWT.RIGHT, 80); //$NON-NLS-1$
+        column.setOptions(new ReportingPeriodColumnOptions(Messages.ColumnDividendYieldOnCost + " {0}", // $NON-NLS-1$ //$NON-NLS-1$
+                        List.of(new SinceX(LocalDate.now().minusDays(365)))));
```

Comment:
Why 365 days instead of one year? It’s not the same (there are leap years).

Reply:
Yes. I look into the ReportingPeriod class and found no 1 Year/12M period. I can define a new one to make it more clear... (and correct in case of a leap year)?

Reply:
fixed now